roofit/doc/developers/test_statistics.md

```diff
@@ -150,8 +178,69 @@ ConstantTermsOptimizer::enableConstantTermsOptimization(&pdf, &normSet, dataset,
 ```
 We refer to RooFit documentation for more about "tracking optimization" which can be enabled or disabled using the final boolean parameter.
 
+## Load balancing options
+
+A number of calculation strategy options are available to tune load balancing.
+Depending on the exact pdf, the effect of these options can be quite dramatic.
+The ideal to strive for is to distribute tasks over worker CPUs equally in terms of total amount of work.
+Since tasks can vary in length, this means all workers simultaneously have something to do and they all finish at the same time.
+The following options using `RooFit::MultiProcess::Config` should all be set before creating the `RooMinimizer` that will perform the fit.
+
+
+### Number of workers
+The most basic setting to tweak is the number of workers $N_w$ using `RooFit::MultiProcess::Config::setDefaultNWorkers`.
+By default, the number of workers is set equal to the number of CPUs.
+Possibly, in some situations it may be better to decrease this number by one or two.
+This is because ``RooFit::MultiProcess`` uses two more processes: the master and the queue processes, giving a total of $N_w + 2$.
+This means two cores by default will have to host two processes, which may slow down the workers on those cores a bit.
```

Comment:
Is there really such a strict mapping between processes and cores? Don't all N cores somehow dynamically share the N + 2 processes as scheduled by the OS?

Reply:
It depends on the OS (and perhaps the hardware, I didn't test this on ARM CPUs for instance). On Linux, the processes get pinned to the cores. This boosts performance a lot, quite critical. On macOS, manual pinning is not possible, so one indeed would have to hope the OS schedules advantageously. The performance decreases if processes switch, because lowest level cache memory has to be reloaded.

Reply:
Ok, thanks for the detailed explanation!